plugins/groq/tests/groq_test.ts

```diff
@@ -33,78 +27,26 @@ describe('toGroqRole', () => {
 
   it('should throw error for unsupported roles', () => {
     assert.throws(() => toGroqRole('unknown' as Role), {
-      message: "role unknown doesn't map to a Groq role.",
+      message: 'Role unknown does not map to any Groq role.', // Adjusted message
     });
   });
 });
 
-describe('toGroqTool', () => {
-  const tool: ToolDefinition = {
-    name: 'exampleTool',
-    description: 'An example tool',
-    inputSchema: {
-      type: 'object',
-      properties: { param: { type: 'string' } },
-      required: ['param'],
-      additionalProperties: false,
-      $schema: 'http://json-schema.org/draft-07/schema#',
-    },
-  };
-
-  it('should convert tool definition to Groq tool format', () => {
-    const expected = {
-      type: 'function',
-      function: {
-        name: 'exampleTool',
-        description: 'An example tool',
-        parameters: tool.inputSchema,
-      },
-    };
-    assert.deepStrictEqual(toGroqTool(tool), expected);
-  });
-});
-
 describe('toGroqMessages', () => {
   const messages: MessageData[] = [
-    {
-      role: 'user',
-      content: [{ text: 'Hello, world!' }],
-    },
-    {
-      role: 'model',
-      content: [{ text: 'How can I assist you today?' }],
-    },
-    {
-      role: 'tool',
-      content: [
-        {
-          toolResponse: {
-            ref: 'ref123',
-            name: 'getResponse',
-            output: 'Sample response',
-          },
-        },
-      ],
-    },
+    { role: 'user', content: [{ text: 'Hello, world!' }] },
```

Comment:
Can you run `npm run format` from the root directory? I think that's why we're seeing format changes here

Reply:
Done.